SIMID/examples/maps/player/simid_player.html

```diff
@@ -0,0 +1,117 @@
+ <html>
```

Comment:
You shouldn't need any of the player/ files.

You can continue to use the player hosted from the simid repo, and only host your creatives here

Reply:
deleted them